frontend/javascripts/admin/task/task_list_view.tsx

```diff
@@ -336,174 +506,8 @@ function TaskListView({ initialFieldValues }: Props) {
-            dataIndex="status"
-            key="status"
-            render={(status, task: APITask) => (
-              <div className="nowrap">
-                <span title="Pending Instances">
-                  <PlayCircleOutlined className="icon-margin-right" />
-                  {status.pending}
-                </span>
-                <br />
-                <span title="Active Instances">
-                  <ForkOutlined className="icon-margin-right" />
-                  {status.active}
-                </span>
-                <br />
-                <span title="Finished Instances">
-                  <CheckCircleOutlined className="icon-margin-right" />
-                  {status.finished}
-                </span>
-                <br />
-                <span title="Annotation Time">
-                  <ClockCircleOutlined className="icon-margin-right" />
-                  {formatSeconds((task.tracingTime || 0) / 1000)}
-                </span>
-              </div>
-            )}
-            filters={[
-              {
-                text: "Has Pending Instances",
-                value: "pending",
-              },
-              {
-                text: "Has Active Instances",
-                value: "active",
-              },
-              {
-                text: "Has Finished Instances",
-                value: "finished",
-              },
-            ]}
-            onFilter={(key, task: APITask) => task.status[key as unknown as keyof TaskStatus] > 0}
-          />
-          <Column
-            title="Edit Position / Bounding Box"
-            dataIndex="editPosition"
-            key="editPosition"
-            width={150}
-            render={(__, task: APITask) => (
-              <div className="nowrap">
-                {formatTuple(task.editPosition)} <br />
-                <span>{formatTuple(task.boundingBoxVec6)}</span>
-              </div>
-            )}
-          />
-          <Column
-            title="Experience"
-            dataIndex="neededExperience"
-            key="neededExperience"
-            sorter={Utils.localeCompareBy<APITask>((task) => task.neededExperience.domain)}
-            width={250}
-            render={(neededExperience) =>
-              neededExperience.domain !== "" || neededExperience.value > 0 ? (
-                <Tag>
-                  {neededExperience.domain} : {neededExperience.value}
-                </Tag>
-              ) : null
-            }
-          />
-          <Column
-            title="Creation Date"
-            dataIndex="created"
-            key="created"
-            width={200}
-            sorter={Utils.compareBy<APITask>((task) => task.created)}
-            render={(created) => <FormattedDate timestamp={created} />}
-            defaultSortOrder={"descend"}
-          />
-          <Column
-            title="Action"
-            key="actions"
-            width={170}
-            fixed="right"
-            render={(__, task: APITask) => (
```

Comment:
The previous code seems to have used two underscores to avoid the shadowing. Imo using  `_unused` is more explicit and easier to spot than using two underscores to avoid the name shadowing. Therefore, the `_unused`  approach is more explicit and therefore imo more preferable.